### PR TITLE
Reverting meeting.ts to v1 state

### PR DESCRIPTION
**Pull request by lakhveerk**
1. Reverting meeting.ts to v1 state
2. Making necessary changes in test-app and unit tests'

### CODE REVIEW

**Review by shrshindeMSFT — DISMISSED** [review]

File: change/@microsoft-teams-js-32d4d6a0-f255-43dd-b341-877c0d63abf9.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Updated meeting namespace to work as it did in v1 along with necessary changes to unit tests and teams-test-app",
```
Comment (erinha) [inline change/@microsoft-teams-js-32d4d6a0-f255-43dd-b341-877c0d63abf9.json:3]: Beautiful! 💯

File: packages/teams-js/test/public/meeting.spec.ts [hunk 2]
```diff
@@ -305,123 +387,173 @@ describe('meeting', () => {
   });
 
   describe('getLiveStreamState', () => {
+    const allowedContexts = [FrameContexts.sidePanel];
+    it('should fail when called without a callback', () => {
```
Comment (erinha) [inline packages/teams-js/test/public/meeting.spec.ts:391]: nit: can you change this test case to say 'should fail when called  with a null callback'.

It's harder to distinguish between "without a callback" and "empty callback", IMO.

File: packages/teams-js/test/public/meetingLegacy.spec.ts [hunk 3]
```diff
@@ -135,16 +135,16 @@ describe('meeting_V1', () => {
```
Comment (erinha) [inline packages/teams-js/test/public/meetingLegacy.spec.ts:135]: As discussed offline, it looks like meeting.spec.ts is a superset of this file after the changes to meeting.ts. I think this file can be deleted.

File: packages/teams-js/src/public/meeting.ts [hunk 4]
```diff
@@ -1,17 +1,9 @@
...
-  callCallbackWithSdkErrorFromPromiseAndReturnPromise,
-} from '../internal/utils';
 import { FrameContexts } from './constants';
 import { SdkError } from './interfaces';
-import { runtime } from './runtime';
 
-/**
- * @alpha
- */
 export namespace meeting {
```
Comment (erinha) [inline packages/teams-js/src/public/meeting.ts:7]: Similar to the [comments in the media.ts PR](https://github.com/OfficeDev/microsoft-teams-library-js/pull/1101#discussion_r855583974), can you please add @deprecated tags to the namespace and APIs referencing version 2.0.0-beta.5?

**Review by erinha — CHANGES_REQUESTED** [review]
Few comments about the test files and @deprecated comments.

**Comment by shrshindeMSFT** [comment]
LGTM!

**Review by shrshindeMSFT — APPROVED** [review]

**Review by erinha — APPROVED** [review]
Looks good!

**Pull request merged**